server/game/cards/events/03/evenhandedjustice.js

```diff
@@ -0,0 +1,65 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class EvenHandedJustice extends DrawCard {
+
+    canPlay(player, card) {
+        if(player !== this.controller || this !== card) {
+            return false;
+        }
+
+        return (_.every(this.game.getPlayers(), this.hasStandingCharacters)
+                && super.canPlay());
+    }
+
+    hasStandingCharacters(player) {
+        return (player.cardsInPlay
+                .filter(card => card.getType() === 'character' && !card.kneeled)
+                .length > 0);
+    }
+
+    play(player) {
+        var waitingPrompt = 'Waiting for opponent to use ' + this.name;
+
+        this.toKneel = [];  // characters to kneel (later)
+
+        // kneel opponent character
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a standing character controlled by your opponent',
+            waitingPromptTitle: waitingPrompt,
+            cardCondition: card =>
+                !card.kneeled
+                && card.getType() === 'character'
+                && card.controller !== this.controller,
+            onSelect: (player, card) => this.toKneel.push(card),
+        });
+
+        // kneel controller character
+        this.game.promptForSelect(player, {
```

Comment:
Hmm, interesting.  Yeah I think we should stick with waiting for one thing to happen rather than queuing two at once.  Could cause interesting side effects if the prompt is cancelled for any reason